### PR TITLE
Remove aria-hidden from scrollable navigation

### PR DESCRIPTION
**Pull request by DenisHdz**
**Overall change:**
Apple has released a new Accessibility feature with macOS Catalina and iOS 13, called Voice Control, that allows you to control your device entirely just with your voice. 

Unfortunately, when applying `aria-hidden: true` to an element, this can't be reached using Voice Control speaking an action, such as "Tap Home".

For this reason, we should remove the `aria-hidden` from the navigation and also the `tab-index:-1` from the links as everything will be visible now to the users.

**Code changes:**
- Do not apply `aria-hidden:true` to the scrollable navigation
- Do not apply `tab-index:-1` to links
- Merge `CanonicalScrollableNavigation` and `AmpScrollableNavigation` into `ScrollableNavigation`
- Remove unnecessary tests
- Remove `isScrollable` prop
- Remove hook from stories
- Remove README references to `aria-hidden`

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx [hunk 1]
```diff
@@ -42,30 +42,13 @@ const StyledScrollableNav = styled.div`
...
-
-  return (
-    <StyledScrollableNav dir={dir} {...ariaHidden}>
-      {React.Children.map(children, child =>
-        React.cloneElement(child, { isScrollable }),
-      )}
-    </StyledScrollableNav>
-  );
-};
+export const CanonicalScrollableNavigation = ({ children, dir }) => (
```
Comment (tochwill) [inline packages/components/psammead-navigation/src/ScrollableNavigation/index.jsx]: This is now exactly the same as the Amp scrollable nav right? And we probably need the spread props for canonical too for event tracking? Should we just merge them?

**Review by OlgaLyubin — COMMENTED** [review]
"When to use this component" and "Amp Menu Button Usage" sections in the Readme still have mentions of the `AmpScrollableNavigation`

**Review by tochwill — APPROVED** [review]

**Comment by tochwill** [comment]
Thanks Denis!

**Review by AlistairGempf — APPROVED** [review]
Looks good to me

**Comment by PriyaKR** [comment]
Looks good to me.Tested the canonical nav with voice over and voice control.

**Pull request merged**